ports/esp8266/modules/ntptime.py

```diff
@@ -7,14 +7,16 @@
 except:
     import struct
 
+from time import localtime
+
 # (date(2000, 1, 1) - date(1900, 1, 1)).days * 24*60*60
-NTP_DELTA = 3155673600
+__NTP_DELTA = 3155673600
 
 # The NTP host can be configured at runtime by doing: ntptime.host = 'myhost.org'
-host = "pool.ntp.org"
+__HOST = "time.google.com"
```

Comment:
The comment here explains how to configure the host name, so there's no need to make this change.